### PR TITLE
Add shyRozen to owners file

### PR DESCRIPTION
**Pull request by shyRozen**

### CODE REVIEW

File: frontend/packages/ceph-storage-plugin/OWNERS [hunk 1]
```diff
@@ -5,11 +5,13 @@ reviewers:
   - cloudbehl
   - gnehapk
   - rawagner
+  - shyRozen
```
Comment (cloudbehl) [inline frontend/packages/ceph-storage-plugin/OWNERS]: Can you create `OWNERS` file inside the integration-tests folder and give access to reviewers and Approvers over there.

**Comment by afreen23** [comment]
Welcom @shyRozen :)

**Comment by cloudbehl** [comment]
Do you also want to do same for noobaa plugin?

**Comment by shyRozen** [comment]
> Do you also want to do same for noobaa plugin?
@cloudbehl I can do it also.

**Pull request merged**